cppcache/src/DistributedSystemImpl.cpp

```diff
@@ -81,9 +81,9 @@ void DistributedSystemImpl::logSystemInformation() const {
 
   ACE_utsname u;
   ACE_OS::uname(&u);
-  LOGCONFIG(
-      "Running on: SystemName=%s Machine=%s Host=%s Release=%s Version=%s",
-      u.sysname, u.machine, u.nodename, u.release, u.version);
+  LOGCONFIG("Running on: SystemName=" BOOST_PLATFORM
```

Comment:
A really important distinction here is that `BOOST_PLATFORM` is resolved at compile time, while `ACE_OS::name` is resolved at runtime. What are you thinking for the other components here? Roll our own system independent solution like `ACE_OS::uname`? When I looked into this I couldn't find a different path.

Reply:
Thanks for pointing that out. Well as you say. Writing our own solution seems like the only way :S

Reply:
Unless I'm missing something, this makes a slight difference in the log output at CONFIG level and above, but otherwise has no effect on behavior of the client, since the data in question isn't used.  So how much do we care about, say, not having as much detail in Windows logs?

Reply:
You've got a point. I did not want to change the output, but it's true that this kind of info is not much relevant and if needed for troubleshooting it can be asked to the issue reporter. If you are OK with removing the call it would make my life a whole lot more easy and future code more maintable. What you think @pivotal-jbarrett ?

Reply:
It is not uncommon for the person collecting the logs to have little knowledge of the host details so I suspect support staff will not be happy if the are working with a customer who can't tell them what OS and version a log came from.  In general I agree though that this is ephemeral information. It is also pretty easy to create the platform independent. You can look to the ACE implementation for hints on how to do the same.

Reply:
That's understandable. Yes, I was planning on re-using ACE implementation, so I'll stick to that!

Reply:
There it is. I've written an implementation for at least all the supported platforms. I've left out some old Windows versions, like Windows 95/98/Me, but I really doubt someone manages to run geode on those OS.

Reply:
Well, considering those are 32bit OSs, out of support and not compatible with our minimum runtime, I think we are ok!